src/Components/Main/TodoForRender/TodoForRender.tsx

```diff
@@ -0,0 +1,161 @@
+import {
+  memo,
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import cn from 'classnames';
+import { useTodoContext } from '../../../Context/Context';
+import { TodoItem } from '../TodoItem/TodoItem';
+import { Todo } from '../../../types/Todo';
+
+export const TodoForRender = memo(() => {
+  const [todoForEdit, setTodoForEdit] = useState<Todo | null>(null);
+  const [editTitle, setEditTitle] = useState('');
```

Comment:
The name is a bit confusing as sounds like a function. Consider using some other name